### ISSUE TITLE
NTR: Ultrasound: ejaculatory duct dilated  (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8702]
New term request Ultrasound: ejaculatory duct dilated  (elthat_211204024211): The duct that delivers sperm into the urethra, adding secretions from the prostate  UMLS: 5542356

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000471
- label: Dilated ejaculatory duct
- parents: [Abnormal male internal genitalia morphology (HP:0000022)](https://hpo.jax.org/app/browse/term/HP:0000022)
- pmid: 
- def: Increase in diameter of the ejeculatory duct. The ejaculatory duct delivers sperm into the urethra, adding secretions from the prostate and is formed by the union of the vas deferens with the duct of the seminal vesicle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8702
- synonyms: Ejaculatory duct dilated(NONE)
